app/mainAppWindow/index.js

```diff
@@ -276,6 +276,7 @@ function onDidFinishLoad() {
   // Inject browser functionality
   injectScreenSharingLogic();
   injectNotificationLogic();
+  injectBrowserLogic();
```

Comment:
I don't think we need this as mention in the other files. Can you check without it and see if it does work realiable?

Reply:
try using the same pattern as the trayWindow in https://github.com/IsmaelMartinez/teams-for-linux/blob/main/app/browser/preload.js#L170 instead of doing the injecting here. That other file does handle stuff like waiting until the UI has all correct etc. Having a separated injection logic (that also starts the activity manager) might create timing issues.